src/app/LegacyConfig.php

```diff
@@ -103,8 +103,12 @@ private static function loadConfig()
         $config['enable_email_contact'] = isset($g_enable_email_contact) && $g_enable_email_contact;
         $config['include_service_body_admin_on_emails'] = isset($include_service_body_admin_on_emails) && $include_service_body_admin_on_emails;
         $config['change_depth_for_meetings'] = $change_depth_for_meetings ?? 0;
-        $config['meeting_states_and_provinces'] = $meeting_states_and_provinces ?? [];
-        $config['meeting_counties_and_sub_provinces'] = $meeting_counties_and_sub_provinces ?? [];
+        $config['meeting_states_and_provinces'] = isset($meeting_states_and_provinces) && is_string($meeting_states_and_provinces)
```

Comment:
We first check if setting is string and then explode on `,` else we default back to setting and either way an empty array if not set or empty sting